Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN \
       libssl-dev \
       libncurses-dev \
       doxygen \
-      libcurl4-openssl-dev \
```

Comment:
I think this is needed by elastic search plugin?

Reply:
Indeed. The line removed was a dublicate, see [here](https://github.com/bitshares/bitshares-core/pull/839/files/172c758c8e89c34e3a2107f9f13b879e148dc973#diff-3254677a7917c6c01f55212f86c57fbfR15)